ts/navigation/params/AppParamsList.ts

```diff
@@ -91,12 +91,13 @@ export type AppParamsList = {
   [IDPayUnsubscriptionRoutes.IDPAY_UNSUBSCRIPTION_MAIN]:
     | NavigatorScreenParams<IDPayUnsubscriptionParamsList>
     | IDPayUnsubscriptionNavigatorParams;
-  [IDPayPaymentRoutes.IDPAY_PAYMENT_CODE_SCAN]: undefined;
+  [IDPayPaymentRoutes.IDPAY_PAYMENT_CODE_SCAN]: undefined; // FIXME: remove after react-navigation 6.x upgrade. This should be insde IDPAY_PAYMENT_MAIN
   [IDPayPaymentRoutes.IDPAY_PAYMENT_MAIN]: NavigatorScreenParams<IDPayPaymentParamsList>;
   [IdPayCodeRoutes.IDPAY_CODE_MAIN]: NavigatorScreenParams<IdPayCodeParamsList>;
 
   [WalletOnboardingRoutes.WALLET_ONBOARDING_MAIN]: NavigatorScreenParams<WalletOnboardingParamsList>;
   [WalletPaymentRoutes.WALLET_PAYMENT_MAIN]: NavigatorScreenParams<WalletPaymentParamsList>;
+  [WalletPaymentRoutes.WALLET_PAYMENT_BARCODE_SCAN]: undefined; // FIXME: remove after react-navigation 6.x upgrade. This should be insde WALLET_PAYMENT_MAIN
```

Comment:
Same as above

Reply:
Added in https://github.com/pagopa/io-app/pull/5130/commits/c43a57b571feaa7222f66b7b8861f2d28b01f3d9